examples/diamond_exit_code/gantry.def.yml

```diff
@@ -0,0 +1,25 @@
+version: "0"
+
+steps:
+  a:
+    image: alpine
+    command: "echo \"Step a\""
+  b:
+    after:
+      - a
+    image: alpine
+    entrypoint: "/bin/sh"
+    command: ["-c", "echo \"Step b\" && exit 1"]
```

Comment:
Hmm, I find it quite confusing that the command's exit code is completely ignored even though it is non-zero and there is a different exit code in the env. Shouldn't at least the commands exit code be forwarded if it isn't provided in the env? That then should be used in the example

Reply:
It's now forwarded but can be replaced with the one set in `gantry.env.yml`.

Reply:
That's one reason to have it, yes. This allows other scripts to handle different types of errors for recovery and notice.